packages/sequelize-model/test/migrations/20191105141900_createGovernmentTable.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
```

Comment:
if I understand right, this file is now redundant, as we have the migrations defined on root package level